src/main/java/com/minecolonies/coremod/proxy/CommonProxy.java

```diff
@@ -154,4 +156,11 @@ private int getNextEntityId()
     {
         return nextEntityId++;
     }
+
+    @Override
+    public File getSchematicsFolder()
+    {
+       return null;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this line start at column 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)